ftw/slider/browser/slider.pt

```diff
@@ -19,10 +19,11 @@
                    tal:content="structure python:images.tag(alt='', title='')"
                    />
             <div class="sliderText"
-                 tal:condition="pane/text">
-              <p tal:condition="pane/title" tal:content="pane/title"
+                 tal:condition="python: pane.show_title or pane.text">
+              <p tal:condition="pane/show_title" tal:content="pane/title"
                   class="documentFirstHeading title"/>
-              <span tal:replace="structure pane/text/output" />
+              <span tal:condition="pane/text"
```

Comment:
This means that in the `setup.py` of `ftw.sliderblock` I will need to require something like:

```
install_requires=[
        'ftw.simplelayout [contenttypes] >= 1.12.0',
        'ftw.slider >= 3.1.0',  # <--- THIS
        'setuptools',
        'plone.dexterity',
        'plone.app.dexterity',
        'Plone',
    ],

```